backend/src/settings.py

```
@@ -1,17 +1,95 @@
+    model_config_path: str = Field(
+        default="model_config.yaml",
+        description="Path to model configuration YAML file"
+    )
+    
+    # Computed fields
+    models: Dict[str, Dict[str, Any]] = Field(default_factory=dict)
+    
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+        self._load_model_config()
+    
```

Ruff (0.12.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)